src/index.html

```diff
@@ -412,6 +415,19 @@ <h2 id="in-their-own-words">In their own words</h2>
       </blockquote>
     </div>
   </div>
+	<div class="container">
+		<h2 id="alternatives">Alternatives to overlays</h2>
+		<p>Since we don't recommend overlays, what do we recommend instead? Here are some options, beginning with free and low-effort and increasing in committment.</p>
+		<ul>
+			<li>Use one of the <strong>free automated accessibility checkers</strong> and make the fixes it recommends for your site. This will only fix about 30% of the potential accessibility problems with your site, but it will not cost you anything except time and it will make improvements for several groups of people without creating problems for others. Most of the free automated audits include useful advice on how to fix the problems they find, but there are also many blog posts and tutorials available online.</li>
+			<li>Use one of the <strong>free accessibility tools</strong> which include a guided manual test as well as an automated check, and make the fixes it recommends for your site. You may not become 100% WCAG compliant but you will be able to address the majority of problems, which will make your site usable for a wide range of people with disabilities.</li>
+			<li><strong>Find or build a widget which exposes more user options</strong> and add that to your site. Giving your users more options to control the personalization of your site is useful for many people who are not tech savvy. Being able to easily increase contrast or text size is useful. Tools like the Fluid UI Preference Editor supports this and other customizations, but you will likely need to employ a developer. To make it look intentional, might be good to hire a designer as well. These can only address some accessibility problems, and definitely are not sufficient on their own.</li>
```

Comment:
I don't think links to specific tools should be added here and am very reluctant about the addition of this line overall. 

Adding the principle of "consider creating or adding a user preferenced toolbar" is ok, but I think this could really confuse product owners. 

My clients mostly want to include ReciteMe accessibility overlay because of the additional tools it offers rather than because they think it solves all accessibility issues. When I ask them what it offers which the browser or operating system does not already, they cite: easy translation, reading aloud, font changes, text size changes. 

**I worry that this may be interpreted by some as a recommendation of accessibility overlays.**